### PR TITLE
Fix tests to allow spaces in "Sass: Create Reusable CSS with Mixins"

### LINKED ISSUE #40212: In challenge : "Sass: Create Reusable CSS with Mixins" doesn't validate with space around ":" character
**Describe your problem and how to reproduce it:**

If you write the mixin properly but you include space in your declarations. For example : 
-moz-border-radius : $radius

The validation system doesn't take it into account. Despite the fact that it works on the side where you can see the result.

**Add a Link to the page with the problem:**
https://www.freecodecamp.org/learn/front-end-libraries/sass/create-reusable-css-with-mixins

**Tell us about your browser and operating system:**
* Browser Name: Firefox
* Browser Version: 82.0
* Operating System: Ubuntu 20.04.1 LTS

**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**
fcc_mixin

### PR DESCRIPTION
**Pull request by lumosan**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #40212

Allows tests to pass when there are spaces around `:` or `;`.

### CODE REVIEW

**Review by naomi-lgbt — DISMISSED** [review]
LGTM~! 🥳 

Congratulations on your first pull request to this project! We look forward to future contributions!

Please note that our workflow requires one additional approval before this can be merged.

File: curriculum/challenges/english/03-front-end-libraries/sass/create-reusable-css-with-mixins.md
Comment (Sky020) [inline curriculum/challenges/english/03-front-end-libraries/sass/create-reusable-css-with-mixins.md:58]:
```suggestion
    testString: assert(__helpers.removeWhiteSpace(code).match(/-webkit-border-radius:\$radius;/gi));
```

File: curriculum/challenges/english/03-front-end-libraries/sass/create-reusable-css-with-mixins.md
Comment (Sky020) [inline curriculum/challenges/english/03-front-end-libraries/sass/create-reusable-css-with-mixins.md:60]:
```suggestion
    testString: assert(__helpers.removeWhiteSpace(code).match(/-moz-border-radius:\$radius;/gi));
```

File: curriculum/challenges/english/03-front-end-libraries/sass/create-reusable-css-with-mixins.md
Comment (Sky020) [inline curriculum/challenges/english/03-front-end-libraries/sass/create-reusable-css-with-mixins.md:62]:
```suggestion
    testString: assert(__helpers.removeWhiteSpace(code).match(/-ms-border-radius:\$radius;/gi));
```

File: curriculum/challenges/english/03-front-end-libraries/sass/create-reusable-css-with-mixins.md
Comment (Sky020) [inline curriculum/challenges/english/03-front-end-libraries/sass/create-reusable-css-with-mixins.md:64]:
```suggestion
    testString: assert(__helpers.removeWhiteSpace(code).match(/border-radius:\$radius;/gi).length == 4);
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
This becomes redundant, when whitespace is removed.

Remember, you can use the _merge batch_ feature in GitHub to accept multiple changes

**Comment by lumosan** [comment]
I didn't know about the merge batch feature, but I think it's done

**Review by ShaunSHamilton — APPROVED** [review]
This LGTM 👍 

Thanks, for making those changes. @nhcarrigan If you are still happy with this, feel free to merge.

**Review by naomi-lgbt — APPROVED** [review]
LGTM~! 🥳 

Congratulations on your first pull request to this project! We look forward to future contributions!

**Pull request merged**
